rtsp-to-web/rootfs/etc/cont-init.d/01-rtsp-to-web

```diff
@@ -32,6 +36,9 @@ cat ${config} \
     | jq ".server.debug = ${debug}" \
     | jq ".server.log_level = \"${log_level}\"" \
     | jq ".server.http_port = \":${http_port}\"" \
+    | jq ".server.ice_servers = \":${ice_servers}\"" \
```

Comment:
Looks like setting ':' in front isn't quite right.  Also, i suspect the syntax for setting a value in a list using jq may be different?

Reply:
Yeah, I haven't been able to test this with the devcontainer on macOS and docker as passing through the docker socket seems to fail. I will spin up a linux VM and try that instead and test this. Thanks for the heads up

Reply:
Correct you don't need the colon.